app/services/aptrust/aptrust_service.rb

```diff
@@ -0,0 +1,350 @@
+# frozen_string_literal: true
+
+module Aptrust
+
+  # TODO: tracking of status of deposit
+
+  class AptrustInfo
+
+    mattr_accessor :aptrust_info_debug_verbose, default: true
+
+    # mattr_accessor :default_description, default: 'This bag contains all of the data and metadata related to a Monograph which has been exported from the Fulcrum publishing platform hosted at https://www.fulcrum.org. The data folder contains a Fulcrum manifest in the form of a CSV file named with the NOID assigned to this Monograph in the Fulcrum repository. This manifest is exported directly from Fulcrum's heliotrope application (https://github.com/mlibrary/heliotrope) and can be used for re-import as well. The first two rows contain column headers and human-readable field descriptions, respectively. {{ The final row contains descriptive metadata for the Monograph; other rows contain metadata for Resources, which may be components of the Monograph or material supplemental to it.}}'
+    mattr_accessor :default_description, default: 'The Description' # TODO
+
+    attr_accessor :access
+    attr_accessor :creator
+    attr_accessor :description
+    attr_accessor :item_description
+    attr_accessor :storage_option
+    attr_accessor :title
+
+    def initialize( work:,
+                    access: nil,
+                    creator: nil,
+                    description: nil,
+                    item_description: nil,
+                    storage_option: nil,
+                    title: nil )
+
+      @access = access; @access = attr_init( @access, 'Institution' )
+      @creator = creator; @creator = attr_init( @item_description, work.creator.join( " & " ) )
+      @description = description; @description ||= default_description
+      @item_description = item_description; @item_description = attr_init( @item_description, work.description.join( " " ) )
+      @storage_option = storage_option; @storage_option = attr_init( @storage_option, 'Standard' )
+      @title = title; @title = attr_init( @title, Array( work.title ).join( " " ) )
+    end
+
+    def attr_init( attr, work_attr )
+      attr ||= work_attr
+      if attr.blank?
+        attr = ''
+      else
+        attr = attr.squish[0..255]
+      end
+      return attr
+    end
+
+    def build
+      <<~INFO
+        Title: #{title}
+        Access: #{access}
+        Storage-Option: #{storage_option}
+        Description: #{description}
+        Item Description: #{item_description}
+        Creator/Author: #{creator}
+      INFO
+    end
```

Comment:
If we're following a similar serialization pattern for `bag-info.txt`, I wonder if storing the key names as strings with the value and then writing a generic serialization method writing out the key-value pairs would be appropriate. Just a thought.